pwa/src/templates/sourcesTemplate/SourcesTemplate.module.css

```diff
@@ -1,10 +1,15 @@
+.container {
+  width: 100%;
+}
+
 .container > :not(:last-child) {
   margin-block-end: var(--gateway-ui-size-2xl);
 }
 
 .section {
   display: flex;
   justify-content: space-between;
+  gap: var(--gateway-ui-size-lg);
   margin-block-end: var(--gateway-ui-size-xl);
 }
 
```

Comment:
The table is now mobile friendly but the Title and add button are not.
You can fix this by simply adding a `@media only screen and (min-width: 576px) {}` in your css and edit the `.section` and `.buttons`